README.md

```diff
@@ -34,146 +34,123 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
+```
 
-#### Object Storage
-1. Create an object storage instance in the cluster
+Use `kubectl` to list pods in the rook namespace. You should be able to see the following: 
 
-   ```bash
-   ./rook object create
-   ```
+```
+$ kubectl -n rook get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+mon0                            1/1       Running   0          1m
+mon1                            1/1       Running   0          1m
+mon2                            1/1       Running   0          1m
+osd-n1sm3                       1/1       Running   0          1m
+osd-pb0sh                       1/1       Running   0          1m
+osd-rth3q                       1/1       Running   0          1m
+rgw-1785797224-9xb4r            1/1       Running   0          1m
+rgw-1785797224-vbg8d            1/1       Running   0          1m
+rook-api-4184191414-l0wmw       1/1       Running   0          1m
+rook-operator-349747813-c3dmm   1/1       Running   0          1m
+```
 
-2. Create an object storage user
+#### Create Storage class
+Before Rook can start provisioning storage, a StorageClass needs to be created. This is used to specify the storage privisioner, parameters, admin secret and other information needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.
+Rook already creates a default admin and demo user, whose secrets are already specified in the sample rook-storageclass.yaml. 
 
-   ```bash
-   ./rook object user create rook-user "A rook rgw User"
-   ```
+However, before we proceed, we need to specify the Ceph monitor IPs. You can find them by running this line (you will need `jq`).
 
-3. Get the connection information for accessing object storage
+```
+$ kubectl -n rook get pod mon0 mon1 mon2 -o json|jq .items[].status.podIP
+"10.2.1.68"
+"10.2.2.66"
+"10.2.0.41"
+``` 
 
-   ```bash
-   eval $(./rook object connection rook-user --format env-var)
-   ```
+Add these IPs into the `monitors` param of the `rook-storageclass.yaml`. Also append the port `6790` to each IPs. They need to be comma-separated.
 
-4. Use an S3 compatible client to create a bucket in the object store
+Create Rook Storage class:
 
-   ```bash
-   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ kubectl create -f rook-storageclass.yaml
+```
 
-5. List all buckets in the object store
+#### Run sample app
 
-   ```bash
-   s3cmd ls --no-ssl --host=${AWS_ENDPOINT} --host-bucket=
-   ```
+Now that rook is running and integrated with Kubernetes, we can create a sample app to verify it. We will create the classic wordpress and mysql apps.
+Both these apps will make use of block volumes provisioned by rook.
 
-6. Upload a file to the newly created bucket
+Start mysql and wordpress:
 
-   ```bash
-   echo "Hello Rook!" > /tmp/rookObj
-   s3cmd put /tmp/rookObj --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ kubectl create -f mysql.yaml
```

Comment:
might want a pointer to these yaml files since the readme isn't in the same folder anymore